target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8072-ax9000.dts

```diff
@@ -121,6 +122,19 @@
 
 	pinctrl-0 = <&i2c_pins>;
 	pinctrl-names = "default";
+
+	emc2305@2f {
+		compatible = "microchip,emc2305";
+		reg = <0x2f>;
+		model_name = "emc2305";
+		#address-cells = <0x01>;
+		#size-cells = <0x00>;
+		emc2305_fan@0 {
+			reg = <0>;
+			name = "emc2305_fan";
+			pwm-output-mode = <1>;
```

Comment:
These properties are useless, please refer to this:
https://github.com/torvalds/linux/blob/master/Documentation/devicetree/bindings/hwmon/microchip%2Cemc2305.yaml